### PR TITLE
Adds redirect after not found exception.

### PR DESCRIPTION
**Pull request by linkouth**
Redirect to home page in case of any error

### CODE REVIEW

**Review by usersadvocate — APPROVED** [review]

File: frontend/src/app/core/interceptors/error-handler.interseptor.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { Observable, throwError } from 'rxjs';
 import { catchError } from 'rxjs/operators';
 
 import { NotificationService } from '../../shared/services';
+import { Router } from '@angular/router';
```
Comment (MatvienkoDO) [inline frontend/src/app/core/interceptors/error-handler.interseptor.ts]: move it to top to imports from angular and rxjs

File: frontend/src/app/core/interceptors/error-handler.interseptor.ts [hunk 2]
```diff
@@ -35,7 +37,7 @@ export class ErrorInterceptor implements HttpInterceptor {
         } else {
           this.notificationService.error('Error: unknown error');
         }
-
+        this.router.navigate(['']);
```
Comment (MatvienkoDO) [inline frontend/src/app/core/interceptors/error-handler.interseptor.ts:41]: it would be nice to check whether we are already here or not

**Review by MatvienkoDO — DISMISSED** [review]

File: frontend/src/app/core/interceptors/error-handler.interseptor.ts (hunk 2 above)
Comment (MatvienkoDO) [reply frontend/src/app/core/interceptors/error-handler.interseptor.ts:41]: In case of network error on home page it will go in cycle
!!!

**Review by MatvienkoDO — CHANGES_REQUESTED** [review]
Comments above

File: frontend/src/app/core/interceptors/error-handler.interseptor.ts (hunk 2 above)
Comment (linkouth) [reply frontend/src/app/core/interceptors/error-handler.interseptor.ts:41]: There is no circle redirect. `router` doesn't allow redirecting on the same route.

**Review by MatvienkoDO — APPROVED** [review]

**Pull request merged**